### PR TITLE
[#] Initial set up

### PR DESCRIPTION
**Pull request by mjordonez2001**
### What ?
- Renders a simple `<div>` with `Hearty Capsules` as text
- Configures ESlint and Prettier
- Configures Jest
- Configures Playwright

### CODE REVIEW

**Comment by mjordonez2001** [comment]
I'm not sure how to get the playwright tests to be successful if I'm running the app locally. Do I have to deploy the app in order for that to work?

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```
Comment (solomonhawk) [inline .github/workflows/playwright.yml:6]: You can probably get rid of `master` and just have `branches: main` (I forget if you need the brackets if you just have 1 branch)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Hearty Capsules
 
 ### Personal Project
-Supplements web app
\ No newline at end of file
+
+Supplements web app
```
Comment (solomonhawk) [inline README.md:5]: Making note here of the Node version you're developing with is important for (hypothetical) other contributors but also future you.

I always recommend https://asdf-vm.com/ for version management, which would mean making a `.tool-versions` file with a `nodejs` line listed.

File: playwright.config.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+
+import { devices } from "@playwright/test";
+import type { PlaywrightTestConfig } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```
Comment (solomonhawk) [inline playwright.config.ts:10]: You could probably get rid of this until later (if you end up needing to read from a `.env` file).

**Review by solomonhawk — APPROVED** [review]
👍  LGTM, would love to see ESLint and prettier jobs in CI.

**Review by nick-telsan — APPROVED** [review]

**Pull request merged**